packages/babel-plugin/src/__tests__/css-builder.test.tsx

```diff
@@ -19,4 +10,43 @@ describe('css builder', () => {
 
     expect(actual).toInclude('@media screen{._43475scu{color:red}._1yzygktf{font-size:20px}}');
   });
+
+  it('generates the correct style prop for shadowed runtime identifiers', () => {
+    const actual = transform(`
+      import '@compiled/react';
+
+      const getBackgroundColor = (color) => color;
+      const color = baseColor;
+
+      <div css={{
+        backgroundColor: getBackgroundColor(customBackgroundColor),
+        color
+      }} />
+    `);
+
+    // Make sure color is used over customBackgroundColor
+    expect(actual).toIncludeMultiple(['{color:var(--_1ylxx6h)}', '"--_1ylxx6h": ix(color)']);
```

Comment:
Hey @MonicaOlejniczak , just for my understanding, without the fix in `evaluateExpression`, we would have the same colour for both `backgroundColor` and `color`? 
For example, if `customBackgroundColor` is yellow, we would have yellow applied on both properties?